pkgs/top-level/all-packages.nix

```diff
@@ -2616,6 +2616,8 @@ in
 
   massren = callPackage ../tools/misc/massren { };
 
+  mat2 = with python3.pkgs; toPythonApplication mat2;
```

Comment:
Fails to evaluate: https://gist.github.com/GrahamcOfBorg/f7ebfc5af0d22b9c6447546f7c51dd9a

Reply:
That's a mystery to me.

Reply:
I think I got it: `kdialog` is an alias for `plasma5Packages.kdialog`.